packages/contracts-bedrock/test/kontrol/scripts/common.sh

```diff
@@ -3,12 +3,18 @@
 
 notif() { echo "== $0: $*" >&2 ; }
 usage() {
-  echo "Usage: $0 [-h|--help] [container|local|dev]" 1>&2
-  echo "Options:" 1>&2
+  echo "Usage: $0 [-h|--help] [container|local|dev] [script|TESTS]" 1>&2
+  echo "" 1>&2
   echo "  -h, --help         Display this help message." 1>&2
+  echo "" 1>&2
+  echo "Execution modes:"
   echo "  container          Run in docker container. Reproduce CI execution. (Default)" 1>&2
   echo "  local              Run locally, enforces registered versions.json version for better reproducibility. (Recommended)" 1>&2
   echo "  dev                Run locally, does NOT enforce registered version. (Useful for developing with new versions and features)" 1>&2
+  echo "" 1>&2
+  echo "Tests executed:"
+  echo "  script             Execute the tests recorded in run-kontrol.sh" 1>&2
+  echo "  TESTS              Execute the tests provided as arguments" 1>&2
```

Comment:
Perhaps the `usage` function should remain in `common.sh`, since `usage` can be accessed from `make-deployment-summary.sh` and `run-kontrol.sh` via `parse_first_arg` (some refactoring needed atm in  `make-deployment-summary.sh`, since right now doesn't accept the `help` argument). We can have specialized `usage` functions for each script and a general one that calls the correct `usage_*` by reading a flag of where the `usage` call comes from. Do you have something else in mind?

Reply:
That works for me!